gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=app.cash.better.dynamic.features
-VERSION_NAME=0.5.1-SNAPSHOT
+VERSION_NAME=0.5.0-agp8.9.2
```

Comment:
I was aiming to bump the patch `0.5.1-agp8.9.2` since there was changes to BDF codesource also with two merged PR? cc @dellisd

Reply:
Ya good point, looks like we haven't run into this yet with the new versioning structure. Makes sense for this to be 0.5.1-agp8.9.2

Reply:
yep, all releases since we changed the versioning have been dependency bumps